packages/scaffold-ui/src/views/w3m-all-wallets-view/index.ts

```diff
@@ -28,6 +44,20 @@ export class W3mAllWalletsView extends LitElement {
     this.onDebouncedSearch(event.detail)
   }
 
+  private onClick() {
+    if (this.badge === 'certified') {
+      this.badge = undefined
+
+      return
+    }
```

Comment:
Why ?

Reply:
I wanted to try doing `this.badge = 'none'`, but the problem is that we won't get non certified and certified wallets, we only would get certified.